lizmap/modules/lizmap/lib/Request/WFSRequest.php

```diff
@@ -634,14 +634,29 @@ protected function getBboxSql($params)
             }
         }
 
+        $srid = $this->qgisLayer->getSrid();
+        if (array_key_exists('srsname', $params)) {
+            $srsname = $params['srsname'];
+            if (!empty($srsname)) {
+                // SRSNAME parameter is not empty
+                // extracting srid
+                $srsname_id = array_pop(explode(':', $srsname));
+                if (ctype_digit($srsname_id)) {
+                    $srid = intval($srsname_id);
+                } else {
+                    return '';
```

Comment:
I do not understand what happens in this case ? Why not letting $srid stay the QGIS layer srid instead ?

Reply:
Because the parameter srsname is false. So I do like false bbox.